src/zoid/checkout/checkout.jsx

```diff
@@ -52,42 +51,33 @@ export const Checkout = create({
     ).render(dom({ doc }));
   },
 
+
```

Comment:
you didn't include customer token prop inside props object, it's required to open customer flow if tracker doesn't include it.

Reply:
I don't think we are supporting the `customer` flow through this for now - just payment flow for guests and users.

we should think about how to support address token though

Reply:
will make a ticket for this